account_move_stripe_import/models/account_journal.py

```diff
@@ -0,0 +1,121 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Akretion (http://www.akretion.com).
+# @author Sébastien BEAU <sebastien.beau@akretion.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+from odoo.addons.account_move_base_import.parser.parser import\
+    AccountMoveImportParser
+from datetime import date
+from collections import defaultdict
+from odoo.tools import DEFAULT_SERVER_DATETIME_FORMAT
+from datetime import datetime
+
+import logging
+_logger = logging.getLogger(__name__)
+
+try:
+    import stripe
+except ImportError:
+    _logger.debug('Can not import stripe')
+
+
+class AccountJournal(models.Model):
+    _inherit = 'account.journal'
+
+    import_type = fields.Selection(selection_add=[('stripe', 'Stripe')])
+
+    @api.model
+    def run_import_stripe_deposit(self):
+        for journal in self.search([('import_type', '=', 'stripe')]):
+            journal.multi_move_import(None, None)
+
+
+class StripeParser(AccountMoveImportParser):
+
+    def __init__(self, journal, *args, **kwargs):
+        super(StripeParser, self).__init__(journal, *args, **kwargs)
+        self.env = journal.env
+
+    @classmethod
+    def parser_for(cls, parser_name):
+        return parser_name == 'stripe'
+
+    def _get_account(self):
+        return self.env['payment.acquirer'].search(
+            [('provider', '=', 'stripe')])
```

Comment:
Domain should be improved to support multi-company : 

[('provider', '=', 'stripe'), ('company_id', '=', self.env.user.company_id.id)])